### PR TITLE
replace BitConverter

### PR DESCRIPTION
**Pull request by Liaojinghui**
Replace `BitConverter.GetBytes` in `Murmur.cs` to `BinaryPrimitives.WriteUInt64LittleEndian`.

### CODE REVIEW

File: src/neo/Cryptography/Murmur128.cs [hunk 1]
```diff
@@ -104,8 +104,10 @@ protected override byte[] HashFinal()
             H2 += H1;
 
             var buffer = new byte[16];
-            Array.Copy(BitConverter.GetBytes(H1), 0, buffer, 0, 8);
-            Array.Copy(BitConverter.GetBytes(H2), 0, buffer, 8, 8);
+            Span<byte> bytes = buffer;
+
+            BinaryPrimitives.WriteUInt64LittleEndian(bytes.Slice(0, 8), H1);
```
Comment (shargon) [inline src/neo/Cryptography/Murmur128.cs:109]: Could you check if it's faster?

Comment (Liaojinghui) [reply src/neo/Cryptography/Murmur128.cs:109]: > Could you check if it's faster? 

This is not for the consideration of efficiency, but the big-little endian issue. This is inspired by the review from Erik on GetNonce.

Comment (shargon) [reply src/neo/Cryptography/Murmur128.cs:109]: But the murmur 128 specification it's in Little or big endian?

Comment (Liaojinghui) [reply src/neo/Cryptography/Murmur128.cs:109]: Little-endian matches the test cases.

**Review by shargon — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**